src/components/Frame.js

```diff
@@ -2,14 +2,26 @@ import React from 'react';
 import classNames from 'classnames';
 import './Frame.css';
 
-const Frame = ({ imageSrc, playing, togglePlaying }) => (
+const Frame = ({
+  imageSrc,
+  playing,
+  togglePlaying,
+  caption,
+  fontColor,
+  textPosition
+}) => (
   <div
     className={classNames('Frame', { 'Frame-empty': !imageSrc })}
     data-testid="Frame-container"
   >
     {imageSrc && (
       <>
-        <img src={imageSrc} alt="current frame" />
+        <div className="frame-container">
+          <img src={imageSrc} alt="current frame" />
+          <p className={textPosition} style={{ color: fontColor }}>
```

Comment:
This class name should not only indicate the `textPosition` but also have a distinguishing class name itself. 

```className={`Frame-container-text ${textPosition`}```

Reply:
Done.